mbed_targets/__init__.py

```diff
@@ -1,4 +1,29 @@
-"""Exposes the primary interfaces for the library."""
+"""mbed-targets provides an abstraction layer for boards and platforms supported by Mbed OS.
 
+This package is intended for use by developers of Mbed OS.
```

Comment:
Sorry one more tiny thing:
```suggestion
This package is intended for use by developers using Mbed OS.
```